bpf/bpf_host.c

```diff
@@ -1096,19 +1096,6 @@ do_netdev(struct __ctx_buff *ctx, __u16 proto, const bool from_host)
 
 #ifdef ENABLE_IPSEC
 	if (!from_host) {
-#ifdef ENABLE_ENCRYPTED_OVERLAY
```

Comment:
Do we want to delete `ENABLE_ENCRYPTED_OVERLAY` from c and `EnableIPSecEncryptedOverlay` from go?

Reply:
Not yet. 

We only removed the EncryptedOverlay specificity for decrypt side. 

When we move to install VinE traffic permanently, we can remove all references to EncryptedOverlay related macros/constants.